.gitlab-ci.yml

```diff
@@ -31,13 +31,15 @@ test:unit:
 
 test:govendor-check:
   variables:
-    GOLANG_VERSION: "1.18"
+    GOLANG_VERSION: '1.18'
```

Comment:
we use double quotes almost everywhere with `GOLANG_VERSION`, I have no idea why.

Reply:
this is just due to the file using majority single quotes (see e.g. the includes right above) - so this just makes it consistent with the rest of the file...
specifically to avoid such considerations there are automatic formatting tools - maybe it would be wise to adopt them!